scripts/buildApiDocs/buildReference.ts

```diff
@@ -70,7 +70,7 @@ export async function buildReference() {
     const descriptionData: PropsTranslations = JSON.parse(descriptionJsonContents);
 
     for (const prop in props) {
-      props[prop].description = descriptionData.propDescriptions[prop]?.description;
+      props[prop].description ??= descriptionData.propDescriptions[prop]?.description;
```

Comment:
Fixes when a `description` is specified in `overrides/*.json` but not reflected in the generated one

Reply:
(Also probably only happens to something that is not literally an "override" like `children`)